exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,8 @@
-var numberOfStudents = 15;
-var numberOfMentors = 8;
+let numberOfStudents = 15;
+let numberOfMentors = 8;
+let sPercentage =(numberOfStudents * 100) / 23;
```

Comment:
Imagine if I tell you to change the number of students to 20, the percentage would have been 87% instead of 71%. Can you think of a better way to calculate the percentage so that it's always correct no matter how many students and mentors are in the variable?